roles/confluent.common/molecule/default/molecule.yml

```diff
@@ -0,0 +1,31 @@
+---
+driver:
+  name: docker
+platforms:
+  - name: centos7
```

Comment:
Is centos8 supported?

Reply:
yes it is... i suppose it wouldn't take anymore time to put a fourth container here... i think we also support ubuntu18